docs/overrides/partials/tabs.html

```diff
@@ -31,7 +31,8 @@
       {% for nav_item in nav %}
         {% include "partials/tabs-item.html" %}
       {% endfor %}
-      <li class="md-tabs__item cloud-link" style="float: right;"><a class="md-tabs__link" target="_blank" rel="noopener noreferrer" href="https://app.prefect.cloud"><b>Go to Prefect Cloud &#187;</b></a></li>
+      <li class="md-tabs__item cloud-filler"></li>
+      <li class="md-tabs__item cloud-link"><a class="md-tabs__link" target="_blank" rel="noopener noreferrer" href="https://app.prefect.cloud"><b>Go to Prefect Cloud &#187;</b></a></li>
```

Comment:
If you wanted to avoid the extra list item, you could add an auto left margin to `.cloud-link` like this:

```css
.cloud-link {
  margin-left: auto;
}
```

which would let you remove the `.cloud-filler` class + element

Reply:
That's a far better solution. Thank you.